src/store/modules/order/actions.ts

```diff
@@ -0,0 +1,45 @@
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import OrderState from './orderState'
+import * as types from './mutation-types'
+import { OrderService } from '@/services/OrderService'
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+
+const actions: ActionTree<OrderState, RootState> = {
+  async fetchDraftOrder ({ commit }, id) {
+    const payload = {
+      draftOrderId: id,
+      shopifyConfigId: "SH_NN_CONFIG"
```

Comment:
This will be configurable from user state.

Reply:
Stored shopifyConfigId in env file for now.